test/unit/test-SqlString.js

```diff
@@ -191,6 +219,15 @@ test('SqlString.format', {
     assert.equal(sql, "'a' and 'b'");
   },
 
+  'question marks are replaced with escaped set values': function() {
+    if (!iterableSupport) return;
+    var set = new Set();
+    set.add('a');
+    set.add('b');
+    var sql = SqlString.format('? and ?', set);
```

Comment:
Wait, what? I don't think this makes sense. I don't think a top-level `Set` object should act like an array, because isn't a `Set` order independent? The top-level is very much order-dependent.